app/src/main/res/layout/link_id_with_patient_view.xml

```diff
@@ -79,7 +79,7 @@
           android:layout_weight="1"
           android:backgroundTint="@color/blue1"
           android:gravity="center"
-          android:text="@string/linkidwithpatient_button_add"
```

Comment:
Can we not rename the resource ID? We can just change the string resource, Transifex handles this case. There does not need to be a 1:1 mapping between the name of the resource and the content.

Reply:
If I don't remove the string from the translations, will the Transifex still handle that case? 🤔

Reply:
Yes, it will be marked as requiring update since the source has changed. The current translations will still be "Next", but that's perfectly fine.